model-optimizer/mo/utils/custom_replacement_config.py

```diff
@@ -399,12 +398,18 @@ def load_and_validate_json_config(config_file_name: str):
     :param config_file_name: name of the file to read from.
     :return: A dictionary serialized from json config file.
     """
-
     try:
         with open(config_file_name, 'r') as f:
             json_config = json.load(f)
-            validator = json_validate.compile(schema_dict)
-            validator(json_config)
+            try:
+                import fastjsonschema as json_validate
```

Comment:
If the fastjsonschema is not installed then we should consider that the JSON file is valid. But current solution will fail the conversion.

Reply:
My bad. Please, disregard the comment